engineering/documents/README.tmpl.md

```diff
@@ -0,0 +1,53 @@
+```
```

Comment:
Also maybe we can provid guides for format, like witch headers, should be used.

Reply:
I think the headers that appear on this list are the ones I'd like people to use

`# Project Name` being the first one